api/http/handler/metrics/handler.go

```diff
@@ -0,0 +1,33 @@
+package metrics
+
+import (
+	"github.com/google/gops/agent"
+	"github.com/gorilla/mux"
+	"github.com/portainer/portainer/api/http/security"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/sirupsen/logrus"
+)
+
+// Handler is the HTTP handler used to handle Prometheus metrics operations.
+type Handler struct {
+	*mux.Router
+}
+
+// NewHandler creates a handler to manage settings operations.
+func NewHandler(bouncer *security.RequestBouncer) *Handler {
+	h := &Handler{
+		Router: mux.NewRouter(),
+	}
+	h.Handle("/metrics", promhttp.Handler())
```

Comment:
who uses this endpoint? the agent?

Reply:
mostly, I keep putting it in so i can make sure i'm not making the performance worse. If I had more time, i'd be using this to measure the impact of the async changes on both time and bandwidth use...

Reply:
oh to actually answer the question - Prometheus, or Grafana cloud agent.